server/sonar-web/src/main/js/apps/issues/sidebar/TagFacet.js

```diff
@@ -75,17 +77,17 @@ export default class TagFacet extends React.PureComponent {
     );
   };
 
-  handleSelect = (tag: string) => {
+  handleSelect = (tag /*: string */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)